frontend/src/components/session/login_form.js

```diff
@@ -49,11 +49,15 @@ class LoginForm extends React.Component {
   }
 
   renderErrors() {
+    const errors = this.props.errors
```

Comment:
you can do this with
```js
const { errors } = this.props;
```
if you want to save yourself from writing variables out twice in the future